css/style.css

```
@@ -356,6 +356,13 @@ img {
 	width: 93px;
 }
 
+#mobile-menu {
+	display: block;
+	overflow: hidden;
+	max-height: 0;
+	transition: max-height 0.2s ease;
```

```suggestion
	transition: max-height 0.35s ease;
```

Making this same as Bootstrap, which is less harsh on phone I find. nbd but trusting the giants here :D